packages/replicache/src/persist/persist.ts

```diff
@@ -65,6 +66,7 @@ export async function persistDD31(
   mutators: MutatorDefs,
   closed: () => boolean,
   formatVersion: FormatVersion,
+  _getZeroData: ZeroOption['getTxData'] | undefined,
```

Comment:
to be used in the next PR. Getting `persist` to work requires changes to `LazyRead` and `LazyStore` which wouldn't fit well to be reviewed in this context.